kubev2v/forklift
# backport ocp boot order

- Skip LUN task progress
- Fix controller SCC namespace
- Add missing mounts for virt-v2v container
- Remove usages of ioutil because it's deprected as of go 1.16
- Fix LUN warm migration
- validation: fix openstack shared disk validation
- plan: check error returned by GetPopulatorTaskName
- ocp: use source client where needed
- Exit when virt-v2v cold hit an error
- Change the storage mapping for OVA
- OCP: add support for secrets and configmaps
- fix virt-v2v bracket
- ocp: use namespace from Ref when present
- ocp: make export token TTL configurable
- openstack: use raw format for images
- backport https://github.com/kubev2v/forklift/pull/497
- Fix virt-v2v entrypoint script
- ocp: fix mapping name check
- Fix TLS SSL connection to FIPS enabled systems
- add -layer suffix to ems layer
- ovirt client: Check if snapshot was created sucesfully
- ovirt: Do not wait for timeout when snapshot removal fails
- Refactor snapshot removal
- Add check for snapshot removal
- Update the timeout phase with the start time
- Remove go util wait from bazel
- Replace MustCode with Code and check if the element exists
- ignore events from ovirt without 'code'
- backport https://github.com/kubev2v/forklift/pull/512
- virt-v2v: Build our own fixed appliance on el8
- ocp: preserve boot order for disks
